doc/lua_api.txt

```diff
@@ -1876,8 +1916,20 @@ i.e. players can more quickly click the nodes away instead of holding LMB.
 
 List of damage for groups of entities. See [Entity damage mechanism].
 
-Example definition of the capabilities of a tool
-------------------------------------------------
+### Punch attack uses (tools only)
+
+Determines how many uses (before breaking) the tool has when dealing damage
+to an object, when the full punch interval (see above) was always waited
+out fully.
```

Comment:
The "was always waited out fully" wording bothers me a bit. I'm not sure what the best way to convey this is, but I feel like something along the lines of "assuming the full punch interval (see above) is waited out fully" might flow better, just from being in the same tense as the rest of the sentence?

You'd also have to update the next paragraph in the same way though. I don't know what's actually best here, I just figured I'd mention that it reads odd.